node_modules/gh-series/tests/util.js

```diff
@@ -577,14 +577,13 @@ var assertSubscribeSeries = module.exports.assertSubscribeSeries = function(clie
 
             // Get the events in this serie so we can verify they appear in the user's calendar
             assertGetSeriesEvents(client, id, null, null, null, null, function(events) {
-
                 // Generate a broad window for the user's calendar
                 var start = moment().subtract(30, 'days').format();
                 var end = moment().add(30, 'days').format();
 
                 // If the serie contained events, we generate a window around the serie events
                 // and assert that the events are returned in the user's calendar
-                if (!_.isEmpty(events.results.length)) {
+                if (!_.isEmpty(events.results)) {
```

Comment:
You're right. I'll fix that and I found another spot with bogus start/end handling, so I'll push a commit for that as well.